nutridine/src/screens/macro/MacroPage.js

```diff
@@ -0,0 +1,214 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  RangeSlider,
+  RangeSliderTrack,
+  RangeSliderFilledTrack,
+  RangeSliderThumb,
+  VStack,
+  Text,
+  HStack,
+  Input,
+  useColorModeValue,
+  Card,
+  CardBody,
+  Box,
+} from "@chakra-ui/react";
+import useMealsByMacros from "../../hooks/useMealsByMacros";
+import { useMeals } from "../../contexts/MealsContext";
+
+function AttributeSliders() {
+  const { setMeals } = useMeals();
+
+  const [calories, setCalories] = useState([200, 500]);
+  const [protein, setProtein] = useState([10, 50]);
+  const [carbs, setCarbs] = useState([20, 100]);
+  const [fat, setFat] = useState([5, 35]);
+
+  const [showSliders, setShowSliders] = useState(true);
+  const [shouldFetch, setShouldFetch] = useState(false);
+  const [desiredMacros, setDesiredMacros] = useState({
+    query: "fast food",
```

Comment:
We'll want to leave the query as `""` in this case. Otherwise, nutritionIX will take this into account and limit our results.

I'm guessing you chucked this in there because the query needs to be non-emnpty to fetch real results. If you note the previous comment about the hook regression, fixing the regression will take care of the empty query for you